google/cloud/spanner/connection.h

```diff
@@ -43,6 +48,21 @@ class Connection {
  public:
   virtual ~Connection() = default;
 
+  struct ReadParams {
```

Comment:
Perhaps `Read()` and `ExecuteSql()` comments like those on `Commit()` and `Rollback()`?

Reply:
I was trying to convince @devjgm that we should get rid of (all) the comments here and just defer to `client.h`.  The class-level comment already hints at that, and the existing ones are partial copies of comments in `client.h`.  But I'll go with the will of the people.

Reply:
I'm happy to go in that direction (no comments here) too.